SensorMap.cpp

```diff
@@ -115,4 +117,11 @@ bool SensorMap::blackTile() {
   uint16_t r, g, b, c;
   tcs_.getRawData(&r, &g, &b, &c);
   return (r < 115 && g < 115 && b < 115);
+}
+
+bool SensorMap::silverTile() {  
+  i2c_->tcaselect(3);
+  uint16_t r, g, b, c;
+  tcs_.getRawData(&r, &g, &b, &c);
+  return (r < 254 && g < 254 && b < 254);
```

Comment:
Think properly of these conditions, because any black tile would also be detected as silver tile

Reply:
I changed the values for the moment, I'm going to check the correct values today and I will put them.